### PR TITLE
Return missing script for team page

### PR DESCRIPTION
**Pull request by freevova**
[this](https://github.com/jetthoughts/jetthoughts.github.io/blob/master/team.html#L88) script is not present in generated files at path `_site/javascripts/team.js`. Need to update gulptask which will compress and copy `tams.js` to `_site/js/team.js`.

### CODE REVIEW

**Comment by freevova** [comment]
@EnragedWildkin Added gulp task **team_script**. It copies ./javascripts/team.js to _site/javascripts/team.js. Then team.js called in _site/team.html.  
Please review.

File: gulpfile.js [hunk 1]
```diff
@@ -179,3 +179,9 @@ gulp.task('wpt_pagespeed', function() {
 gulp.task('pagespeed', function() {
   gulp.run('grunt-pagespeed');
 })
+
+gulp.task('team_script', function () {
+  gulp.src(assets.js)
+    .pipe(uglifyjs('team.js'))
+    .pipe(gulp.dest('./_site/javascripts'))
```
Comment (mir4a) [inline gulpfile.js]: @freevova destination should be `./js`

**Comment by mir4a** [comment]
@freevova lets include this script into app.js and add rule if elements `.team_member` is present on page, then turn set height script

File: gulpfile.js (hunk 1 above)
Comment (mir4a) [inline gulpfile.js]: @freevova revert this

**Comment by freevova** [comment]
@mir4a @EnragedWildkin Please review.

File: javascripts/team.js [hunk 2]
```diff
@@ -1,51 +1,52 @@
...
+      heights.push(h);
+    });
+    var max = Math.max.apply(Math, heights);
+
+    els.css('height', max + 'px');
+  }
+
+
+  var team = $('.team_member');
+  sameHeight(team);
```
Comment (EnragedWildkin) [inline javascripts/team.js]: This should not be called unless `team`?

File: javascripts/team.js [hunk 3]
```diff
@@ -1,51 +1,52 @@
...
 
+  if($('.team_member').length)
```
Comment (EnragedWildkin) [inline javascripts/team.js]: Can we use `team` here?

**Comment by EnragedWildkin** [comment]
@freevova Please check my comments

File: javascripts/team.js [hunk 4]
```diff
@@ -1,51 +1,52 @@
...
+      var h = $self.height();
+      heights.push(h);
+    });
+    var max = Math.max.apply(Math, heights);
+
+    els.css('height', max + 'px');
+  }
+
+
+  var team = $('.team_member');
```
Comment (mir4a) [inline javascripts/team.js]: @freevova lets rename this variable to `$team` and use it in if rule to check if we're on team's page

**Comment by freevova** [comment]
@EnragedWildkin Please review.

**Comment by EnragedWildkin** [comment]
@freevova Looks good. 

Please update commit message body to tell "What" and "Why" according to http://chris.beams.io/posts/git-commit/
E.g. Why this change is needed? What was a problem?

**Comment by freevova** [comment]
@EnragedWildkin Please review.

**Pull request merged**